Assets/Scripts/ArrowScript.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ArrowScript : MonoBehaviour
+{
+    Rigidbody arrowRB;
+
+    private void Start()
+    {
+        arrowRB = GetComponent<Rigidbody>();
+    }
+
+    private void OnCollisionEnter(Collision collision)
+    {
+        /*
+         * Not sure what would be the most efficient could be done in one of multible ways
+         * arrowRB.sleep(); but if arrow is hit by another arrow they both fall
+         *
+         */
+        if (!collision.gameObject.CompareTag("arrow"))//To keep players from stacking arrows oddly
+        {
+            arrowRB.isKinematic = true;
+            arrowRB.velocity = Vector3.zero;
```

Comment:
I don't think this is necessary since kinematic rigidsbodys already don't move on their own

Reply:
Yep, now I know. I knew they weren't affected by external forces I didn't know they stopped  moving completely.